easybuild/easyconfigs/o/OpenMolcas/OpenMolcas-22.10-intel-2022a-noGA.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenMolcas'
+version = '22.10'
+versionsuffix = '-noGA'
+
+homepage = "https://gitlab.com/Molcas/OpenMolcas"
+description = """OpenMolcas is a quantum chemistry software package.
+This build does not support MPI and can only be run on a single node."""
+
+toolchain = {'name': 'intel', 'version': '2022a'}
+
+source_urls = ['https://gitlab.com/Molcas/OpenMolcas/-/archive/v%(version)s/']
+sources = ["OpenMolcas-v%(version)s.tar.gz"]
+patches = ['OpenMolcas-21.06_mcpdft_deps.patch']
+checksums = [
+    {'OpenMolcas-v22.10.tar.gz': 'd1d3d6c9bd69486c36f5285a51ac18e75c03540c5b33751855e94abc764d9658'},
+    {'OpenMolcas-21.06_mcpdft_deps.patch': '708763e060aa0174bbe9c0cc336b509fbf492066465908dd919452836a7a793a'},
+]
+
+builddependencies = [('CMake', '3.23.1')]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('HDF5', '1.12.2'),
+]
+
+build_shared_libs = True
+
+configopts = '-DLINALG=MKL -DMKLROOT=$MKLROOT '
+configopts += '-DOPENMP=ON -DTOOLS=ON -DFDE=ON'
+configopts += '-DMPI=OFF -DGA=OFF '
```

Comment:
There is a space missing at the end of the second line, so we get `-DFDE=ON-DMPI=ON`. My suggestion to make sure we avoid this issue:
```py
_copts = [
    "-DLINALG=MKL",
    "-DMKLROOT=$MKLROOT",
    "-DOPENMP=ON",
    "-DTOOLS=ON",
    "-DFDE=ON",
    "-DMPI=OFF",
    "-DGA=OFF",
]
configopts = ' '.join(_copts)
```